pkg/apis/migration/v1alpha1/directvolumemigration_types.go

```diff
@@ -132,6 +151,24 @@ type RsyncOperation struct {
 	Failed bool `json:"failed,omitempty"`
 }
 
+func (r *RsyncOperation) Equal(other *RsyncOperation) bool {
```

Comment:
Wonder if we can refactor this function to 

```
func (x *RsyncOperation) Equal(y *RsyncOperation) bool {
```
When reading equality x on the left and y on the right side of equality makes it easier to read.

Reply:
+1, updating